action.yml

```diff
@@ -25,13 +25,25 @@ inputs:
     description: 'The deployment id of the deployment info from source service'
   deploy-platform:
     required: false
-    description: 'The deployment platform such as CodeDeploy, Spinnaker'
+    description: 'The deployment platform, e.g. CodeDeploy, Spinnaker'
   deploy-app-name:
     required: false
     description: 'The name of the application for deployment info'
   deploy-app-platform:
     required: false
-    description: 'The platform application is being deployed to such as ECS/K8s'
+    description: 'The platform application is being deployed, e.g. ECS/K8s'
+  build-build-id:
```

Comment:
Added these new inputs for deployment. Previously we assumed the current workflow run would be `build` that is associated with the `deployment`. Not sure that we can make that assumption, but this adds a ton of buildKey inputs needed for the deployment to be correctly associated with the build. @cjwooo @tovbinm Thoughts?

Reply:
Oh well, they aren't required, so it's ok.